sdk/identity/azure-identity/src/test/java/com/azure/identity/AzureCliCredentialTest.java

```diff
@@ -77,7 +77,7 @@ public void azureCliCredentialAzNotLogInException() throws Exception {
         // test
         AzureCliCredential credential = new AzureCliCredentialBuilder().build();
         StepVerifier.create(credential.getToken(request))
-            .expectErrorMatches(e -> e instanceof Exception && "Azure  not Login".equals(e.getMessage()))
+            .expectErrorMatches(e -> e instanceof CredentialUnavailableException && e.getMessage().contains("Azure  not Login"))
```

Comment:
Does the actual error message have an additional space? Could we clean that up if so.

```suggestion
            .expectErrorMatches(e -> e instanceof CredentialUnavailableException && e.getMessage().contains("Azure not Login"))
```